whatsapp_theming/lib/darkmode.dart

```diff
@@ -0,0 +1,177 @@
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:theme_provider/theme_provider.dart';
+
+class DarkModeTest extends StatefulWidget {
+  @override
+  _DarkModeTestState createState() => _DarkModeTestState();
+}
+
+class _DarkModeTestState extends State<DarkModeTest> {
+  int radiovalue = 0;
+  void radioValue(int val) {
+    setState(() {
+      radiovalue = val;
+    });
+  }
+
+  saveDarkMode(int value) async {
+    SharedPreferences prefs = await SharedPreferences.getInstance();
+    prefs.setBool('isUserNew', false);
+    prefs.setInt('radiovalue', value);
```

Comment:
`radiovalue` is a key here. `radiovalue` is an int above. There is also a `radioValue()` function. Please use proper naming to differentiate what each is.